native/build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
+	id 'cpp-library'
+	id 'cpp-unit-test'
+	alias(libs.plugins.spotless)
+}
+
+library {
+	targetMachines = [
+			machines.windows.x86, machines.windows.x86_64, machines.windows.architecture("aarch64")
+	]
+}
+
+library {
+	linkage = [
+		Linkage.SHARED
+	]
+}
+
+import org.gradle.internal.jvm.Jvm
+def jvmHome = Jvm.current().javaHome
```

Comment:
Wouldn't it be better to use something like the `java.home` system property or the `JAVA_HOME` env var to avoid Gradle internals?

Reply:
Probably yeah, blame my lazyness and the Jetbrains docs lol: https://www.jetbrains.com/help/idea/setting-up-jni-development-in-gradle-project.html#adding_jni_support